src/SnippetProvider/RaygunSnippetProvider.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace SilverStripe\TagManager\SnippetProvider;
+
+use SilverStripe\Core\Environment;
+use SilverStripe\TagManager\SnippetProvider;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms;
+use SilverStripe\Core\Manifest\ModuleResourceLoader;
+
+/**
+ * A snippet provider that lets you add Raygun
+ */
+class RaygunSnippetProvider implements SnippetProvider
+{
+    public function getTitle()
+    {
+        return "Raygun crash reporting (JavaScript front-end)";
+    }
+
+    public function getParamFields()
+    {
+        $apiKeyDefined = !empty(Environment::getEnv('SS_RAYGUN_APP_KEY'));
+
+        $imgSrc = ModuleResourceLoader::singleton()->resolveURL('sminnee/tagmanager:client/img/raygun-guide.png');
+
+        $fields = [];
+        if ($apiKeyDefined) {
+            $fields[] = (new Forms\CheckboxField('ENV_APP_KEY', 'Use "silverstripe/raygun" module'))
```

Comment:
* Can we just import these FQCNs?
* Please use `::create()` so we get dependency injection.